docs/getting-started/quickstart.rst

```diff
@@ -213,7 +213,7 @@ start a REPL session and follow along:
 
     .. code-block:: irb
 
-      irb(main):005:0> OSO.allowed?(actor: "alice", action: "GET", resource: "expense")
+      irb(main):005:0> OSO.allowed?(actor: alice, action: "GET", resource: "expense")
```

Comment:
super nit: all of these `irb` line numbers should be sequential

currently it goes 001 002 003 004 005 006 005 005 006